docs/upgrade/1-0-0.md

```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+title: Upgrade to 1.0.0
+parent: Upgrade
+nav_order: 3
+---
+
+# Upgrade to 1.0.0
+
+Open Distro for Elasticsearch 1.0.0 uses Elasticsearch 7.0.1, which has numerous breaking changes from 6.x.x. This page includes several important considerations when upgrading from Open Distro for Elasticsearch 0.x.x to 1.x.x.
+
+- Cluster and index names can no longer contain the `:` character.
+- Indices now default to one shard rather than five.
+- `hits.total`, returned as part of search responses, is now an object, not an integer. If you created monitors using the Alerting plugin, you probably need to update them to use the new response format (`hits.total.value` rather than `hits.total`). The old format:
```

Comment:
@mihirsoni let me know how this portion looks.

Reply:
This looks good to me. 
Should we provide step-to-step guide to do this  ? Which will provide easy way to migrate for customers.

Reply:
Sure, when you say step-by-step, do you just mean the following?

1. Open Kibana
1. Alerting -> Monitors -> Select a monitor
1. Select a trigger -> Edit
1. Change any occurrences of `ctx.results[0].hits.total` to `ctx.results[0].hits.total.value`
1. Update

All the visual graph monitors use aggregations, so it's not clear to me if those are affected by this change. Let me know. Thanks!